midterm-2023-11-07/code/5159-ana-ilievska/script.ts

```diff
@@ -0,0 +1,110 @@
+document.addEventListener("DOMContentLoaded", siteCode);
+// function siteCode(){
+//      console.log("Starting"); 
+// }
+
+interface Movie {
+    id:  number;
+    title: string;
+    director: string;
+    year: number;
+    genre: [string];
+    plot: string;
+    cast: Actors[]; 
+    <oscars>(): string;
+    rating: number;
+}
+
+interface Actors {
+    actor: string, 
+    character: string;
+}
+
+async function siteCode () {
+    const data = await loadData();
+    console.log(data);
+    const displayMovies = (data);
```

Comment:
This actually creates a local variable called `displayMovies` and assigns it the value of the data.
It does **not** call the `displayMovies` function with `data` as the parameter.